### DISCUSSION TITLE
Public goods & Quadratic Funding knowledge exchange

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#2946] [General]
We had chat with @kkukelka @petersopko who would like to study public good stuff and came up with an idea to found a board to pooling knowledge on quadratic funding and public goods 

- [David Hawig - Web3 Grants and the future of Public Goods Funding](https://www.youtube.com/watch?v=JcX5urijD0s)
- https://gov.gitcoin.co/t/grants-2-0-is-how-we-scale-our-impact/10015
- https://wtfisqf.com/?grant=&grant=&grant=&grant=&match=1000
- I can update the rest of the links here and scrape from DMs
- feel free to drop yours

**Comment by petersopko** [comment]
I've done some research, dropping this here:

**More Resources:**

- Brand new Public Goods paper from Vitalik et al., May 11th, 2022 [Decentralized Society: Finding Web3's Soul](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4105763) -> previously discussed "soulbound" feature described in it
- GitCoin QF [repo](https://github.com/gitcoinco/quadratic-funding)
- The way GitCoin retroactively dropped tokens for their init. QV [here](https://thedefiant.io/gitcoin-drops-gtc-token-to-25500-users-now-worth-more-than-100m/) -> the process of deploying tokens is interesting
- Another useful paper by Vitalik from 2018 (good read when it comes to philosophy/description of the problem) [Liberal Radicalism: A Flexible Design For Philanthropic Matching Funds](https://blogchains.org/wp-content/uploads/sites/4/2019/04/SSRN-id3243656.pdf)
- "Old institution Experiment" [Colorado Tried a New Way to Vote: Make People Pay—Quadratically](https://www.wired.com/story/colorado-quadratic-voting-experiment/)
- DoraHacks (old HackerLink) example of a proposal on Polkassembly which got (?) rejected, maybe something to learn from the exchange [here ](https://kusama.polkassembly.io/post/488)
- Quadratic Funding V2 Protocol: Anti-Sybil, Fairness, and Scalability On-Chain DoraHacks [medium blog post](https://dorafactory.medium.com/quadratic-funding-v2-protocol-anti-sybil-fairness-and-scalability-on-chain-2a7aec43541c) -> why is on-chain identity needed
- Quadratic Funding Pallet from OAK [here](https://github.com/OAK-Foundation/quadratic-funding-pallet) (seems it's...not implemented anywhere?) -> this is [where](https://github.com/w3f/Grants-Program/pull/268) is their proposal on w3f gh
- Quadratic Funding Palet from DoraHacks -> can't seem 

**Questions/things to figure out:**
(how to get to the Quadratic Funding ideal)

**0. Token Airdrop From Gitcoin** as inspiration, just a quote here:
	The total supply of GTC is 100,000,000 (100M) tokens, broken down as follows.
	-   Retroactive Airdrop – 15%
	-   Gitcoin DAO – 50%
	-   Existing Stakeholders – 35%

**1. KodaDao?**
	- DAO could be the next step for pushing KodaDot's decision (when it comes to development) into truly Public Goods / Decentralized mode. Quadratic Voting could incentivize creators/users to vote on the direction of further development and allocated Q funds would incentivize devs to work on specific features (introducing milestones here?)
	- how to do this properly? Most DAOs use the catchy term but are not at all DAOs
	- implementation -> further research needed. This will be timely once we have settled on the model
	
**2. Retroactive token drop** -> what mechanism should we use to determine the amount of token dropped? inspiration can be found [here](https://gitcoin.co/blog/introducing-gtc-gitcoins-governance-token/)
	- one thing bugs me, why is there this sentence? "And please remember — GTC has no economic value; it is a governance token used to oversee the Gitcoin ecosystem. It has no claim on financial rights."  -> yes and no? The token is being traded, so it has **some** economic value https://www.coingecko.com/en/coins/gitcoin . Since the coin is not bound to  **shares**, that's what they meant?
	
**3. The process to keep the tokens distribution going** -> further research needed

**4. Avoiding Sybil attacks with on-chain identity** -> afaik, getting on-chain identity verified by a validator may be the prerequisite for participating in governance/voting

**5. Voting on new features (functionality > design in the first place)** -> introduction of Milestones. We could effectively do this now; I'd like to move from issues labeled `Chief` into using the Milestones feature of GH. Some of our `Chief`  [issues](https://github.com/kodadot/nft-gallery/issues/2650) have 30+ comments, at which point they are becoming quite intricate for people not included in the initial conversation. The chosen name for this feature, "Milestones," is slightly misleading; the due date is entirely optional. https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones

**Reply by petersopko** [reply]
https://github.com/DoraFactory/Quadratic-Funding-Grant-Pallet
